samples/drumthumper/build.gradle

```diff
@@ -6,7 +6,7 @@ android {
     buildToolsVersion "29.0.1"
 
     defaultConfig {
-        applicationId "com.mobileer.drumthumper"
+        applicationId "com.google.oboe.samples.drumthumper"
```

Comment:
The use of com.mobileer in DrumThumper was deliberate because we plan to release it as an app on Play Store. We cannot use a package name of com.google in the Play Store for apps that are not first party.

Reply:
Sure, but that doesn't mean we need to use `com.mobileer` as the default package name in the project. Doing so causes Android Studio to complain that all the classes have the wrong package name. What we should do is have a 'prod' config in gradle which changes the `applicationId` when we want to create an APK for Play upload. This should be done in a separate PR though.

Reply:
Don says the package names do not matter for Play Store. Lacking evidence to the contrary I will approve this.
@android-usb - What do you think?

Reply:
I am inclined to trust Don on this.